frontend/specs/participant.spec.ts

```diff
@@ -95,7 +95,7 @@ test('launching study and aborting it', async ({ page }) => {
     await page.click('testId=launch-study')
 
     // qualtrics will redirect here once complete
-    await goToPage({ page, path: `/study/land/${studyId}?consent=false`, loginAs: 'user' })
+    await goToPage({ page, path: `/study/land/${studyId}?abort=true`, loginAs: 'user' })
 
```

Comment:
see comments below in study-landing.tsx, I don't think we want to rename this

Reply:
Lack of consent is no longer an abort, so we need a separate way to report an aborted study. My one thought was it might be good for it to report a reason, rather than just a boolean, but we can no longer abort on consent=false